docs/quality-assurance/test-profiles.md

```diff
@@ -1,5 +1,24 @@
-# UKHO Test Engineer Profiles
-Our job roles adhere to the [DDAT](https://www.gov.uk/government/collections/digital-data-and-technology-profession-capability-framework#qat:-test-engineer-) QAT Test Engineer roles. We have detailed the responsibilities of each role:
+# UKHO Test Engineer Profiles, Skill Levels & Responsibilities
+Our job roles adhere to the [DDAT](https://www.gov.uk/government/collections/digital-data-and-technology-profession-capability-framework#qat:-test-engineer-) QAT Test Engineer roles. 
+
+The different levels of Test Engineer, as per DDaT, are: 
```

Comment:
DO you need to say DDAT again, having mentioned it in the last sentence?

Reply:
Updated to reflect your comment :)